manifests/generated/operator-csv.yaml.in

```diff
@@ -876,7 +874,7 @@ spec:
                 - mountPath: /etc/virt-operator/certificates
                   name: kubevirt-operator-certs
                   readOnly: true
-              priorityClassName: kubevirt-cluster-critical
+              priorityClassName: system-node-critical
```

Comment:
Wouldn't `virt-operator` be `system-cluster-critical` and `virt-handler` be the most important one with `system-node-critical`?If `virt-handler` goes away from the node, we loose higher level contact to all VMs there, while even a downtime of a few on `virt-operator` should not be too much of an issue for kubevirt.

Reply:
My thought process with this was that if `virt-operator` goes away all components could become unresponsive since it is what handles cert rotation. As where if `virt-handler` has downtime, while the user would be unable to manage their workloads (stop/start/migrate etc) their actual workloads would still be running. 

But I also think the other way as you said could make sense as well. I guess it depends what we define as the core functionality?

Reply:
Good thoughts. I think the main reason is that `virt-operator` can easily be rescheduled to other nodes. `system-cluster-critical` should make it pretty easy for it to find another spot on another node. `virt-handler` can't go somewhere else.

Reply:
ahh yeah that makes sense. Do you think `virt-api` and `virt-controller` staying at the `system-cluster-critical` level is okay, since they are also deployments and can more easily find another node? so in the end the priority classes should be as follows...

virt-handler -> system-node-critical 

virt-api -> system-cluster-critical
virt-controller -> system-cluster-critical
virt-operator -> system-cluster-critical

Reply:
Rethought this. Considering that we would with these classes potentially compete with e.g. a CNI plugin or a storage daemon, I think it would really be better if we would be somewhere between core components and the user. What do you think?

Reply:
virt-handler I consider to be highest priority in kubevirt because we want to make sure active VMs continued to be served and it can't be moved to another node.  I think the rest of kubevirt can be a lower priority.